wiki/History_of_osu!/2022/de.md

```diff
@@ -0,0 +1,203 @@
+
+Neben anderen Änderungen verwendet osu!mania aufgrund von Abänderungen beim Sternebewertungssystem und dem Performance-Punkte-System jetzt Genauigkeit anstatt den Score bei der Berechnung von Performance-Punkten.[^osu-mania-sr-pp]
+
+osu! überschritt 4.294.967.296 Scores, was bedeutet, dass das alte Datenbanksystem sein Limit erreichte. Jeder Score wird als eine einzigartige Ganzzahl ohne Vorzeichen von der Länge 32 Bit hinterlegt. Das heißt also, dass die Datenbank nicht mehr als 2^32 Scores speichern kann. Die erweiterte Datenbankinfrastruktur war bereits Monate im Voraus geplant und mit einigen Wochen Vorlauf erfolgreich migriert.[^score-limit][^osu-death][^score-submission] Der letzte Score auf dem alten System war [\*namirin - Kanzen Shouri\*Esper Girl](https://osu.ppy.sh/scores/osu/4294967295), gespielt von [ElDanetex](https://osu.ppy.sh/users/18930084).
+
+Kreative Leute stellten ihr Können beim Skinning-Wettbewerb [Chromatic Alteration](https://osu.ppy.sh/home/news/2022-07-18-skinning-contest-chromatic-alteration-announcement) unter Beweis. Die Herausforderung dieses Wettbewerbs hatte eine interessante Abwechslung, denn die zu verwendeten Farben mussten aus einem der drei vorgegebenen Farbspektren sein, die den Sehschwächen [Protanomalie, Deuteranomalie oder Tritanomalie](https://de.wikipedia.org/wiki/Farbenblindheit) entsprachen. Die [Gewinner dieses Wettbewerbs](/wiki/Contests/Skinning_Contest/3#podium) auf dem ersten, zweiten und dritten Platz waren [thetasigma](https://osu.ppy.sh/users/6234482), [Corne2Plum3](https://osu.ppy.sh/users/15646039) und [koomo](https://osu.ppy.sh/users/2168518).[^chromatic-alt-results]
+
+osu!(lazer) erhielt die Mod Freeze Frame, die jeweils nur eine Combo anzeigt, was mehr Lesevermögen von den Spielern verlangt.[^freeze-frame]
+
+Ein Update auf der osu!-Webseite machte es möglich, den höchsten Rang eines Spielers, den er jemals erreicht hat, anzuzeigen, sobald man mit der Maus über den Spielerrang auf dem Profil fuhr.[^highest-rank]
+
+Für das diesjährige Halloween gab es erneut einen [Fanart-Wettbewerb](https://osu.ppy.sh/home/news/2022-10-10-halloween-fanart-contest), bei dem Künstler dazu aufgefordert wurden, die osu!-Maskottchen in gespenstischen Kostümen zu zeichnen. [Dreamxiety](https://osu.ppy.sh/users/13103233) gewann den ersten Platz im Wettbewerb mit 3.749 Stimmen.[^halloween] Eine Einsendung wurde widerrufen, weil sie als zu angsteinflösend angesehen wurde und dem Autor wurde stattdessen ein eigenes Abzeichen verliehen.[^grin-reddit][^grin-twitter][^grin-badge]
```

Comment:
I like gruseligen tbh. Seems to be the best translation for spooky imo.
```suggestion
Für das diesjährige Halloween gab es erneut einen [Fanart-Wettbewerb](https://osu.ppy.sh/home/news/2022-10-10-halloween-fanart-contest), bei dem Künstler dazu aufgefordert wurden, die osu!-Maskottchen in gruseligen Kostümen zu zeichnen. [Dreamxiety](https://osu.ppy.sh/users/13103233) gewann den ersten Platz im Wettbewerb mit 3.749 Stimmen.[^halloween] Eine Einsendung wurde widerrufen, weil sie als zu angsteinflösend angesehen wurde und dem Autor wurde stattdessen ein eigenes Abzeichen verliehen.[^grin-reddit][^grin-twitter][^grin-badge]
```

Reply:
I tried to find a word which carried this atmosphere from the event over as best as possible. What I mean is this feeling of drawing images that are kind of spooky and have these cutesy art styles but are obviously not Outlast-like creepy. I'm okay with your suggestion!